website/src/components/emailSignupButton.tsx

```diff
@@ -80,21 +93,25 @@ export default function EmailSignupButton() {
         </div>
       </div>
       <div className="mx-auto w-max max-w-full text-center text-xs font-normal text-stone-500 md:text-base dark:text-stone-400">
-        {signupNumber === null && (
-          <>
-            Sign up and we&#039;ll let you know when the app is ready!
+        {!hasSignedUp && (
```

Comment:
make sure this is hidden when `submitStatus` is  "loading". There is a bug where it shows the wrong number while loading